watt.py

```diff
@@ -26,8 +26,8 @@
 parser.add_argument("-e", "--executor", help="Path to cromwell jar.", default=os.environ.get("EXECUTION_ENGINE"))
 parser.add_argument("--executor-log-prefix",
                     help="Prefix for cromwell log path; outputs will be [flag_input]-[workflow]-[test_name].log.",
-                    default=sys.stdout)
-parser.add_argument("-c", "--config", help="Test configuration file.", default=CONFIG_DEFAULT_NAME, nargs=1)
+                    default="cromwell ")
```

Comment:
This is probably a good idea. It just means the cromwell logs don't flood stdout instead, which was the default, right? Maybe we should name the default to something like `watt_logs/cromwell`? That sounds a little verbose though. But it seems the default shouldn't be to flood the current dir with a bunch of log files if you run a lot of tests.

Reply:
yeah, I actually changed this because the default ended up crashing later on because the code expects this to be a string which sys.stdout is not.  But I agree, in general flooding stdout or the current dir is probably not the right option is probably not probably not the right option, so I'll switch it to `watt_logs/cromwell`

Reply:
Oh, I was probably just using string values and forgot to even check the default. Thanks!